modules/apps/account/account-test/src/testFunctional/macros/json/accountRole/JSONAccountRoleAPI.macro

```diff
@@ -4,11 +4,12 @@ definition {
 	macro _associateUser(accountRoleId = null, accountEntryId = null, userId = null) {
 		Variables.assertDefined(parameterList = "${accountEntryId},${accountRoleId},${userId}");
 
+		var adminDefaultPassword = JSONUtil2.getAdminUserPassword();
 		var portalURL = JSONCompany.getPortalURL();
 
 		var curl = '''
 			${portalURL}/api/jsonws/account.accountrole/associate-user \
-				-u test@liferay.com:test \
+				-u formatJSONAdminUser() \
```

Comment:
I think this should be inside ${   } and if in this case is the adminDefaultPassword needed? (because here you are using the method to have the full concat